app/src/main/java/net/yslibrary/monotweety/status/ComposeStatusController.kt

```diff
@@ -268,11 +267,12 @@ class ComposeStatusController(private var status: String? = null) : ActionBarCon
     childRouter.popCurrentController()
   }
 
-  @TargetApi(Build.VERSION_CODES.N_MR1)
   fun reportShortcutUsedIfNeeded(status: String?) {
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1 && status.isNullOrEmpty()) {
-      applicationContext?.getSystemService(ShortcutManager::class.java)
-          ?.reportShortcutUsed("newtweet")
+    if (status.isNullOrEmpty()) {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+        @Suppress("NewApi")
+        applicationContext?.getSystemService(ShortcutManager::class.java)?.reportShortcutUsed("newtweet")
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Class requires API level 25 (current min is 19): android.content.pm.ShortcutManager [NewApi]